src/deploy/database/release.ts

```diff
@@ -1,11 +1,9 @@
-"use strict";
+import * as clc from "cli-color";
 
-var clc = require("cli-color");
+import * as rtdb from "../../rtdb";
+import * as utils from "../../utils";
 
-var rtdb = require("../../rtdb");
-var utils = require("../../utils");
-
-module.exports = function (context) {
+export function release(context: any): Promise<any> {
```

Comment:
Same as above - can this be Promise<void>? And optionally, this is cleaner as an async functon.

Reply:
This does return a promise. It just doesn't use `async/await` - is that what you meant?